src/bid/score.rs

```diff
@@ -5,22 +5,105 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 //! Score generation
-#![cfg(feature = "std")]
 
-use super::Score;
-use crate::bid::Bid;
-use crate::errors::BlindBidError;
-use dusk_bls12_381::BlsScalar;
-use dusk_bytes::Serializable;
-use dusk_jubjub::JubJubAffine;
-use dusk_plonk::prelude::*;
-use num_bigint::BigUint;
-use num_traits::{One, Zero};
-use plonk_gadgets::{
-    AllocatedScalar, RangeGadgets::max_bound, ScalarGadgets::maybe_equal,
+cfg_if::cfg_if! {
+    if #[cfg(feature = "canon")] {
+        use canonical::Canon;
+        use canonical_derive::Canon;
+    }
+}
+
+cfg_if::cfg_if! {
```

Comment:
Cannot just use one single `cfg_if` block (but with two `if #` inside?

Reply:
Makes sense

Reply:
I meant using one `cfg-if` block instead of two but as mentioned is not possible. So it's ok as is.